app/src/main/java/com/cherrish/android/presentation/challenge/routine/ChallengeRoutineViewModel.kt

```
@@ -0,0 +1,61 @@
+
+    fun onNextClick() {
+        val state = _uiState.value
+
+        if (state !is UiState.Success) return
+        if (!state.data.isSelected) return
+
+        val selectedRoutine = state.data.selectedRoutine!!
+    }
+
+    fun onBackClick() {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)